client/src/node.rs

```diff
@@ -123,6 +136,49 @@ impl Node {
         Ok(())
     }
 
+    async fn get_payloads(&self, start_slot: u64, end_slot: u64) -> Result<Vec<ExecutionPayload>, NodeError> {
```

Comment:
Can we move this into `ConsensusClient`? Want to have most consensus related stuff separated, so that we can support over evm chains or L2s in the future by swapping out to a new `ConsensusClient`.

Reply:
Sure, good point. I moved the code over accordingly.